third-party/pipelines-as-code/CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# @kubernetes-models/pipelines-as-code
```

Comment:
You don't have to add a changelog manually. Changeset will handle this for you. Just write this in the changeset.